### ISSUE TITLE
ImportError: cannot import name 'DinoAugmentationCropper' from 'sparsam.data_augmentation

### ISSUE THREAD

**Issue by saifhassan** [IPMI-ICNS-UKE/sparsam#3]
Following Error occurs while running `dino_example.py`: 
```
Traceback (most recent call last):
  File "sparsam/dino_example.py", line 14, in <module>
    from sparsam.train import create_dino_gym, StudentTeacherGym
  File "/home/user1/Documents/research@saif/self-supervised/sparsam/sparsam/train.py", line 28, in <module>
    from utils.Dataset import MultiCropDatasetWrapper
  File "/home/user1/Documents/research@saif/self-supervised/sparsam/utils/Dataset.py", line 12, in <module>
    from sparsam.data_augmentation import DinoAugmentationCropper
  File "/home/user1/Documents/research@saif/self-supervised/sparsam/sparsam/data_augmentation.py", line 10, in <module>
    from sparsam.utils import min_max_normalize_tensor
  File "/home/user1/Documents/research@saif/self-supervised/sparsam/sparsam/utils.py", line 19, in <module>
    from sparsam.data_augmentation import DinoAugmentationCropper
ImportError: cannot import name 'DinoAugmentationCropper' from 'sparsam.data_augmentation' (/home/user1/Documents/research@saif/self-supervised/sparsam/sparsam/data_augmentation.py)
```

**Comment by Moxnie** [comment]
I am not able to reproduce this error, are you in general able to import from sparsam i.e. in a plain python console?

**Comment by saifhassan** [comment]
> I am not able to reproduce this error, are you in general able to import from sparsam i.e. in a plain python console?

No, I cannot `import DinoAugmentationCropper` from `sparsam.data_augmentation`. Error occurs in utils.py file.

**Comment by Moxnie** [comment]
Could you please pull the latest version and check if the error still occurs

**Comment by saifhassan** [comment]
In `sparsam/utils.py`, on `Line 341`, `*` with `*data` is typo error in your code or it has some intention like pointers?
```
Traceback (most recent call last):
  File "sparsam/dino_example.py", line 14, in <module>
    from sparsam.train import create_dino_gym, StudentTeacherGym
  File "/home/user1/Documents/research@saif/self-supervised/sparsam/sparsam/train.py", line 30, in <module>
    from sparsam.utils import (
  File "/home/user1/Documents/research@saif/self-supervised/sparsam/sparsam/utils.py", line 341
    return image, *data
                  ^
SyntaxError: invalid syntax
```

**Comment by dericdesta** [comment]
Please upgrade your Python version (the syntax error has been fixed in Python 3.8, cf. [changelog](https://docs.python.org/3.8/whatsnew/changelog.html#:~:text=bpo%2D35814%3A%20Allow%20unpacking%20in%20the%20right%20hand%20side%20of%20annotated%20assignments.%20In%20particular%2C%20t%3A%20Tuple%5Bint%2C%20...%5D%20%3D%20x%2C%20y%2C%20*z%20is%20now%20allowed.))
